cli_reference/helm-cli/getting-started-with-helm-on-openshift-container-platform.adoc

```diff
@@ -6,6 +6,11 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
+[NOTE]
```

Comment:
Please use the module instead: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#indicating-technology-preview-features

Reply:
Thank you for pointing that out! Missed it somehow. Changed.

Reply:
Thank you for pointing that out! Missed it somehow. Changed.